global.json

```diff
@@ -1,7 +1,8 @@
 {
   "sdk": {
-    "version": "8.0.100",
-    "rollForward": "latestFeature"
+    "version": "9.0.100-preview.6.24328.19",
```

Comment:
The version value does not support wild cards. `rollForward` should help pick the latest.